cmsl1t/analyzers/jetMet_analyzer.py

```diff
@@ -230,9 +230,9 @@ def _plots_from_cfgs(self, cfgs, puBins, emulator=False, high_range=False):
 
             thresholds = []
             for l1trig, thresh in allThresholds.items():
-                if l1trig.replace("_Emu", "") in cfg.name:
-                    if emulator and "Emu" not in l1trig:
-                        continue
+                if emulator and "Emu" not in l1trig or not emulator and "Emu" in l1trig:
+                    continue
+                if l1trig.replace("_Emu","") in cfg.name:
```

Comment:
So the only style mistake here is a missing space after the comma (what travis is complaining about):
```python
if l1trig.replace("_Emu","") in cfg.name:
# ->
if l1trig.replace("_Emu", "") in cfg.name:
```